TODO

```diff
@@ -0,0 +1,46 @@
+### Design v0.2.0
+
+### Old Design - v0.1.0
+
+#### Sync CSPCAutoKeeper
+    - watch - CSPCAuto
+    - attachments - BlockDevices (available & annotated)
+    - sync - Create/Apply CSPCAutoKeeper with BlockDevices that are annotated
+
+#### Sync Storage & BlockDevices
+    - watch - CSPCAuto
+    - attachments - BlockDevices (available)
+    - attachments - Storage (annotated)
+    - sync - Iterate BlockDevices:
+        - Build map of BlockDevices with Annotation
```

Comment:
Why is the annotation required on blockdevice? BlockDevice resources are meant to be read only, for all the consumers right.If

Reply:
@akhilerm we may not work on this design.